insights/specs/insights_archive.py

```diff
@@ -122,7 +122,7 @@ class InsightsArchiveSpecs(Specs):
     ls_ocp_cni_openshift_sdn = simple_file("insights_commands/ls_-l_.var.lib.cni.networks.openshift-sdn")
     ls_origin_local_volumes_pods = simple_file("insights_commands/ls_-l_.var.lib.origin.openshift.local.volumes.pods")
     ls_osroot = simple_file("insights_commands/ls_-lan")
-    ls_run_systemd_generator = simple_file("insights_commands/ls_-lan_.run.systemd.generator")
+    ls_run_systemd = simple_file("insights_commands/ls_-lanRL_.run.systemd")
```

Comment:
It would be good to keep compatible with the archives that were already collected with the old spec, please use `first_file` here.
```
ls_run_systemd = first_file("insights_commands/ls_-lanRL_.run.systemd", "insights_commands/ls_-lan_.run.systemd.generator")
```

Thanks.

Reply:
Never mind, since this spec is no longer used by any rules. I think it's OK to change it to match the new command directly.